lib/globals.d.ts

```diff
@@ -1,4 +1,22 @@
+// This will clash with the node 'global' namespace...
```

Comment:
Note - if it doesn't require action, it should be a bit more verbose, because I'm not sure what it's trying to tell me.